lang/he/index.md

```diff
@@ -80,46 +80,46 @@ language_dir: rtl
+זוהי האחריות שלך בתור מפתח/ת מקצועי/ת לתעד את התוכנה שמיועדת לשימוש בידי אחרים. ניהול המורכבות של התכונה הינו חשוב מאין כמוהו לשמירת הפרויקט יעיל, וזה משהו שקשה לעשות אם אף אחד לא יודע כיצד להשתמש בתוכנה שלך, או לאיזה מתודות אפשר לקרוא בבטחה. בטווח הארוך, גרסאות סמנטיות, וההתעקשות על API מוגדר היטב יכול לשמור על כולם ועל הכל פועל בצורה חלקה.
 
 ### מה עלי לעשות אם אני בטעות משחרר שינויים לא תואמים אחורה כגרסה משנית (Minor)?
 
-ברגע שאתה מבין ששברת את מפרט הגירסאות הסמנטיות, תתקן את הבעיה ותשחרר גרסה משנית (Minor) חדשה עם שיחזור התאימות לאחור. אפילו תחת הנסיבות האלו זה בלתי מקובל לשנות גרסאות קודמות. אם זה מתאים למקרה, תתעד את הגרסה הסוררת ותידע את המשתמשים שלך על הבעיה כדי שיהיו מודעים לגרסה זו.
+ברגע שאת/ה מבין/ה ששברת את מפרט הגירסאות הסמנטיות, תקן/י את הבעיה ושחרר/י גרסה משנית (Minor) חדשה עם שיחזור התאימות לאחור. אפילו תחת הנסיבות האלו זה בלתי מקובל לשנות גרסאות קודמות. אם זה מתאים למקרה, תעד/י את הגרסה הסוררת ויידע/י את המשתמשים שלך על הבעיה כדי שיהיו מודעים לגרסה זו.
 
 ### מה עלי לעשות אם אני מעדכן את התלויות שלי מבלי לשנות את ה API הציבורי?
 
-זה יחשב כתואם מכיוון שזה לא משפיע על ה API הציבורי. תוכנה שתלויה באופן מפורש באותם תלויות בהם התוכנה שלך תלויה צריכה לכלול מפרט תלויות משלהם והמחבר של אותה התוכנה יבחין בקונפליקטים. קביעה האם מדובר בשינוי ברמת תיקון (Patch) או ברמה משנית (Minor) תלוי בהאם אתה מעדכן את התלויות שלך כדי לתקן באג או להציג פונקציונליות חדשה. בדרך כלל הייתי מצפה לקוד נוסף במקרה האחרון, ובמקרה כזה זה כמובן מצריך העלה של גרסה משנית (Minor).
+זה יחשב כתואם מכיוון שזה לא משפיע על ה API הציבורי. תוכנה שתלויה באופן מפורש באותן תלויות בהן התוכנה שלך תלויה צריכה לכלול מפרט תלויות משלה והמחבר/ת של אותה התוכנה י/תבחין בקונפליקטים. קביעה האם מדובר בשינוי ברמת תיקון (Patch) או ברמה משנית (Minor) תלוי בהאם את/ה מעדכן/ת את התלויות שלך כדי לתקן באג או להציג פונקציונליות חדשה. בדרך כלל הייתי מצפה לקוד נוסף במקרה האחרון, ובמקרה כזה זה כמובן מצריך העלה של גרסה משנית (Minor).
 
 ### מה אם אני משנה בטעות את ה API הציבורי באופן שאינו תואם עם שינוי מספר הגרסה (למשל קוד מציג שינוי שובר תאימות ראשי (Major) בשחרור בעל שינוי בגרסת התיקון (Patch) בלבד)
 
-השתמש בשיקול הדעת שלך. אם יש לך קהל ענק שיושפע באופן גדול משינוי ההתנהגות אחורה לכוונה המקורית של ה API הציבורי, אז יכול להיות שהכי טוב יהיה לבצע שחרור של גרסה ראשית (Major), אפילו אם התיקון יכול להיחשב כגרסת תיקון (Patch) בלבד. זכור, ניהול גרסאות סמנטי הוא על העברת מסר באיך שמספר הגרסה משתנה. אם השינויים האלה חשובים למשתמשים שלך, השתמש במספר הגרסה כדי ליידע אותם.
+השתמש/י בשיקול הדעת שלך. אם יש לך קהל ענק שיושפע באופן גדול משינוי ההתנהגות אחורה לכוונה המקורית של ה API הציבורי, אז יכול להיות שהכי טוב יהיה לבצע שחרור של גרסה ראשית (Major), אפילו אם התיקון יכול להיחשב כגרסת תיקון (Patch) בלבד. זכור/י, ניהול גרסאות סמנטי הוא על העברת מסר באיך שמספר הגרסה משתנה. אם השינויים האלה חשובים למשתמשים שלך, השתמש/י במספר הגרסה כדי ליידע אותם.
```

Comment:
Should this be "זכור/זכרי" ?

Reply:
I don't think so. He uses the same structure in past sentences, for example:
```
של אותה התוכנה י/תבחין בקונפליקטים.
```
instaed of "יבחין/תבחין". So we should agree on one structure and be consists throughout the text. I think it's more impotent.

Reply:
This is good question - basically there two cases where the "/" is not perfect for both male and female (זכור/י and י/תבחין). @arokem @nirgn975 as you've both highlighted the two options are:
1. Keep the format for consistency, text is understandable despite minor grammatical inaccuracy
2. List the full male/female versions, which breaks consistency but avoids the grammatical inaccuracy.

Personally I have no preference (I believe either would work). I see @nirgn975 favors (1). @arokem, what do you think?

Reply:
I think that the slash can denote either omission/addition of letters to the word, or a replacement of one letter by another, so I have no problem with "י/תבחין" (or "אתם/ן", as another example) but I don't see how this works for "זכור/י". Following this logic, I would only change "זכור/י".  But I leave this to your judgement. It's not a big issue.

Reply:
Thanks @arokem. Seeing as this is an isolated case, I think it would be ok to use the full (זכור/זכרי) in favor of the document being 100% grammatically correct.